app/timetables/models.py

```diff
@@ -1,3 +1,10 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from app.shared.models import CommonInfo
+
+class Course(CommonInfo, models.Model):
+    name = models.CharField(max_length=150)
```

Comment:
@andela-snwuguru please let us make the name field unique, so we will not have duplicate entries in the database.